generated/provider_dependencies.json

```
@@ -26,12 +26,13 @@
       "apache-airflow>=2.4.0",
       "asgiref",
       "asgiref",
-      "boto3>=1.24.0",
+      "boto3>=1.28.0",
+      "botocore>=1.28.0",
```

We could also change lower-bound for `botocore`

```suggestion
      "botocore>=1.31.0",
```

Releases between `boto3` and `botocore` are shifted by 3 minor version, e.g. for boto3==1.28.0 required botocore>=1.31.0,<1.32.0

---

Yeah. I have not been sure about it (noticed the difference). If that's the rule then it's cool

---

I can't say it is official rule, but this what I observed last 3-4 years, since boto-team resolved dependency hell between their packages, the things was much worse than in `aiobotocore` (to be honest [there is a reason](https://aiobotocore.readthedocs.io/en/latest/contributing.html#background-and-implementation) why aiobotocore has this strict dependency)